hack/start-build.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -euo pipefail
+
+oc -n openshift-ingress-operator start-build ingress-operator \
+   ${V+--follow} --wait
+
+if [[ -n "${DEPLOY+1}" ]]
+then
+    oc -n openshift-ingress-operator patch deploy/ingress-operator \
+       --type=strategic --patch='
```

Comment:
Will the CVO allow this? If not, should we be updating the image via the CVO somehow? Would love to get away from disabling the CVO during development

Reply:
CVO will revert the change when it syncs.  Usually I disable CVO if I need it not to revert the image, but I can look into [patching the clusterversion](https://github.com/openshift/cluster-version-operator/blob/3956b767d634e7eab7dc1c322e7393330bd2d641/docs/dev/clusterversion.md#setting-objects-unmanaged) to override the image.  Should that be a separate PR?

Reply:
I think disabling CVO is standard practice for us at this point when it comes to any of the hack tooling — was just curious if you had thought more about it. It would be nice to, at some near point in the future, figure out how to work with the CVO rather than against it in these cases.